src/modules/shared/forms/engine/form-engine.component.ts

```diff
@@ -29,6 +30,7 @@ export class FormEngineComponent implements OnInit, OnChanges {
 
   @Input() parameters: FormEngineParameterModel[] = [];
   @Input() values?: { [key: string]: any } = {};
+  @Output() formChanges = new EventEmitter();
```

Comment:
This should be typed. We should always use typescript to type all communications and interfaces.